content/akeneo-event-platform/concepts.md

```diff
@@ -160,44 +162,46 @@ Example of an event payload for a productDeleted event
 
 For more information, consult the [CloudEvents spec attributes](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md).
 
-## Retry policies
-
-Ensure reliable message processing in distributed systems. Retry policies help manage transient failures and guarantee message delivery even when issues occur.
+## Key platform behaviors
 
-Here, we outline the key **retry policies** implemented in our system.
+Your subscribing service implementation and architecture must deal with the following capabilities and constraints to consume events from the platform at its best.
 
-### ACK timeout
```

Comment:
"ack" terminology is pretty "pubsub" tainted.

changed to the notion of "delivery timeout"